Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -275,7 +381,11 @@ export async function updateAzureRMWebAppConfigDetails(endPoint, webAppName: str
 			deferred.resolve();
 		}
 		else {
-			deferred.reject(response.statusMessage);
+            var errorMessage = "Status Code : " + response.statusCode + "\n";
```

Comment:
We may want to localize it

Reply:
It's just StatusCode and Error, which are standard terms not planning to localize.